changelogs/fragments/update_rmbase.yaml

```
@@ -0,0 +1,7 @@
+---
+minor_changes:
+  - >
+    `NetworkTemplate` and `ResouceModule` base classes have been moved under `module_utils.network.common.rm_base`.
+     Stubs have been kept for backwards compatibility. These will be removed in ansible.netcommon v3.0.0.
```

```suggestion
     Stubs have been kept for backwards compatibility. These will be removed after 2023-01-01.
```
I wouldn't pin this to a netcommon version because we have a process for deprecating things on a schedule, and 3.0.0 may not necessarily line up with that (and I think won't... 3.0.0 is scheduled for June of next year as far as I remember)